Formula/vulkan-validationlayers.rb

```diff
@@ -0,0 +1,68 @@
+class VulkanValidationlayers < Formula
+  desc "Vulkan layers that enable developers to verify correct use of the Vulkan API"
+  homepage "https://github.com/KhronosGroup/Vulkan-ValidationLayers"
+  url "https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/refs/tags/v1.3.247.tar.gz"
+  sha256 "f24b37ada8545389239b7fb3caa5e2dd93acede96c23a471dc8dd8cbae2bcbf9"
+  license "Apache-2.0"
+  head "https://github.com/KhronosGroup/Vulkan-ValidationLayers.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on "googletest" => :build
+  depends_on "python@3.11" => :build
+  depends_on "vulkan-loader" => :test
+  depends_on "vulkan-tools" => :test
+  depends_on "glslang"
+  depends_on "spirv-headers"
+  depends_on "spirv-tools"
+  depends_on "vulkan-headers"
+
+  on_linux do
+    depends_on "libx11" => :build
+    depends_on "libxcb" => :build
+    depends_on "pkg-config" => :build
+    depends_on "wayland" => :build
+  end
+
+  def install
+    args = [
+      "-DGLSLANG_INSTALL_DIR=#{Formula["glslang"].prefix}",
+      "-DSPIRV_HEADERS_INSTALL_DIR=#{Formula["spirv-headers"].prefix}",
+      "-DSPIRV_TOOLS_INSTALL_DIR=#{Formula["spirv-tools"].prefix}",
+      "-DVULKAN_HEADERS_INSTALL_DIR=#{Formula["vulkan-headers"].prefix}",
+      "-DBUILD_LAYERS=ON",
+      "-DBUILD_LAYER_SUPPORT_FILES=ON",
+      "-DBUILD_TESTS=OFF",
+      "-DUSE_ROBIN_HOOD_HASHING=OFF",
+    ]
+    if OS.linux?
+      args += [
+        "-DBUILD_WSI_XCB_SUPPORT=ON",
+        "-DBUILD_WSI_XLIB_SUPPORT=ON",
+        "-DBUILD_WSI_WAYLAND_SUPPORT=ON",
+      ]
+    end
+    system "cmake", "-S", ".", "-B", "build", *args, *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+  end
+
+  def caveats
+    <<~EOS
+      In order to use this layer in a Vulkan application, you may need to place it in the environment with
+        export VK_LAYER_PATH=#{share}/vulkan/explicit_layer.d
+    EOS
+  end
+
+  test do
+    ENV.prepend_path "VK_LAYER_PATH", share/"vulkan/explicit_layer.d"
+    ENV["VK_ICD_FILENAMES"] = Formula["vulkan-tools"].lib/"mock_icd/VkICD_mock_icd.json"
+
+    expected = <<~EOS
+      Instance Layers: count = 1
+      --------------------------
+      VK_LAYER_KHRONOS_validation Khronos Validation Layer #{version}  version 1
+    EOS
+    actual = shell_output("#{Formula["vulkan-tools"].bin}/vulkaninfo --summary")
```

Comment:
```suggestion
    actual = shell_output("vulkaninfo --summary")
```
Let's just get this from `PATH`.